ts-fold.el

```diff
@@ -200,19 +199,19 @@ Return nil if no valid node is found.
 This function is borrowed from `tree-sitter-node-at-point'."
   (let* ((pos (or pos (point)))
          (mode-ranges (alist-get major-mode ts-fold-range-alist))
-         (root (tsc-root-node tree-sitter-tree))
-         (node (tsc-get-descendant-for-position-range root pos pos))
+         (root (treesit-buffer-root-node))
+         (node (treesit-node-descendant-for-range root pos pos))
          ;; Used for looping
          (current node))
-    (while (and current (not (alist-get (tsc-node-type current) mode-ranges)))
-      (setq current (tsc-get-parent current)))
+    (while (and current (not (alist-get (treesit-node-type current) mode-ranges nil t #'equal)))
+      (setq current (treesit-node-parent current)))
```

Comment:
need to use the optional parameters for `alist-get` so we can do an `equal` compare since the keys are strings now.